src/argilla/client/feedback/dataset.py

```diff
@@ -574,6 +574,20 @@ def push_to_argilla(
                         disable=show_progress,
                     ):
                         for record in updated_records[i : i + PUSHING_BATCH_SIZE]:
+                            if not record.id:
```

Comment:
These were defined in `set_suggestions` to avoid the users from calling `set_suggestions` and then after setting suggestions for all the records realising that they cannot push those to Argilla and need to complete the process again. But we can also define it here if you think is better.